static/app/utils/profiling/flamegraph/flamegraphStateProvider/index.tsx

```diff
@@ -280,3 +329,36 @@ export function FlamegraphStateProvider(
     </FlamegraphStateContext.Provider>
   );
 }
+
+type Candidate = {
+  score: number;
+  threadId: number | null;
+};
+
+function scoreFlamegraph(
```

Comment:
What is this scoring for?

Reply:
this relates to the comment below, we assign a score to the flamegraph for each thread to pick one that we want to show